packages/realm/src/app-services/App.ts

```diff
@@ -82,7 +83,7 @@ function toBindingMetadataMode(arg: MetadataMode): binding.MetadataMode {
 const translationTable: Record<binding.MetadataMode, MetadataMode> = {
   [binding.MetadataMode.NoEncryption]: MetadataMode.NoEncryption,
   [binding.MetadataMode.Encryption]: MetadataMode.Encryption,
-  [binding.MetadataMode.NoMetadata]: MetadataMode.NoMetadata,
+  [binding.MetadataMode.InMemory]: MetadataMode.NoMetadata, // TODO: should we rename NoMetadata to InMemory?
```

Comment:
I don't think we should rename unless this will be part of a new next major SDK version.